### PR TITLE
Fix/josh/edit profile cleanup

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/userProfile/components/UserProfileModal.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                    country,
+                    email,
+                    firstName,
+                    lastName,
+                    phoneNumber,
+                    username,
+                },
+            } = this.props;
+
+            const PROFILE_INFO = [
```
Comment (Way2nnadi) [inline src/components/userProfile/components/UserProfileModal.js]: this should not be defined here

**Review by enyeInc — APPROVED** [review]
Looks good enough to me

File: src/components/userProfile/components/UserProfileModal.js (hunk 1 above)
Comment (joshuaobinna) [reply src/components/userProfile/components/UserProfileModal.js]: Ok

**Review by EngrEric — APPROVED** [review]
Looks good to me

**Pull request merged**